Extensions/WindowsServiceReleaseTasks/Src/Tasks/InstallTopshelfService/InstallTopshelfService.ps1

```diff
@@ -35,7 +35,7 @@ Try {
         
         Write-Host ("##vso[task.setvariable variable=E34A69771F47424D9217F3A4D6BCDC95;issecret=true;]$servicePassword")  # Make sure the password doesn't show up in the log.
         
-        $additionalSharedArguments += " -username:$serviceUsername"
+        $additionalSharedArguments += " -username ""$serviceUsername"""
         if (-Not [string]::IsNullOrWhiteSpace($servicePassword)) {
             $additionalSharedArguments += " -password:""$servicePassword"""
```

Comment:
looks ok
in theory, passwords could have spaces in also, so would it be advisable to apply the same change here too ?

Reply:
service password already has that feature.  Someone gonna merge this change?